Retrieved from upstream bug tracker on 2011-02-10:
http://sourceforge.net/tracker/?func=detail&aid=3136392&group_id=21965&atid=373401
Original change 166de50914191303b232ec123cfbaadad521560c in AltLinux git:
http://git.altlinux.org/people/avm/packages/gdmap.git


From: Alexander Myltsev <avm@altlinux.ru>
Date: Wed, 21 Apr 2010 18:43:14 +0000 (+0400)
Subject: GTK_WIDGET_SENSITIVE -> gtk_widget_get_sensitive
X-Git-Tag: gdmap-0.8.1-alt2~1
X-Git-Url: http://git.altlinux.org/people/avm/packages/?p=gdmap.git;a=commitdiff_plain;h=166de50914191303b232ec123cfbaadad521560c;hp=f25dd07708a61ee08e1a8a932ea692c7117f3f5c

GTK_WIDGET_SENSITIVE -> gtk_widget_get_sensitive
---

diff --git a/gdmap/src/gui_support.c b/gdmap/src/gui_support.c
index 3027e29..45272d3 100644
--- gdmap/src/gui_support.c
+++ gdmap/src/gui_support.c
@@ -85,7 +85,7 @@ on_ebox_enter(GtkWidget* box, GdkEventCrossing* event, GtkLabel* label) {
   const char* text;
 
   (void)event;
-  if (GTK_WIDGET_SENSITIVE(label)) {
+  if (gtk_widget_get_sensitive(GTK_WIDGET(label))) {
     char* temp;
 
     text = g_object_get_data(G_OBJECT(label), "label");
@@ -129,7 +129,7 @@ static void on_clear_label(GtkWidget* widget) {
 void ui_event_label_set_sensitive(GtkWidget* ebox, gboolean set) {
   gpointer callback = g_object_get_data(G_OBJECT(ebox), "callback");
   GtkWidget* child = GTK_BIN(ebox)->child;
-  if (GTK_WIDGET_SENSITIVE(child) == set) return;
+  if (gtk_widget_get_sensitive(child) == set) return;
 
   if (set && !callback) return;
   gtk_widget_set_sensitive(child, set);
